### DISCUSSION TITLE
New Search UX

### DISCUSSION THREAD

**Discussion by VishnuSanal** [TeamAmaze/AmazeFileManager#3984] [Show and tell]
There are three modes of searching:

- First, we search the current directory. This happens as the user types.

- Next, an indexed search. Searches the `MediaStore` by filtering items from the current directory and its children.

- Next, a recursive search is performed from the root directory on the file listing (home) screen.

Each mode gets triggered upon pressing "can't find what you are looking for?" button.

<img src="https://github.com/TeamAmaze/AmazeFileManager/assets/50027064/d8bdea4c-aa26-4204-9c70-7038f5ccca8e" height=600 />

**Comment by EmmanuelMess** [comment]
Remember to design the way multiple tabs interact with the search.

**Reply by VishnuSanal** [reply]
> Remember to design the way multiple tabs interact with the search.

It doesn't work this way right now. The results appear in a new screen.
